generated/niscope/enums.py

```diff
@@ -24,6 +24,28 @@ class AcquisitionType(Enum):
     '''
 
 
+class AddressType(Enum):
+    PHYSICAL = 0
+    '''
+    Physical address.
+    '''
+    VIRTUAL = 1
+    '''
+    Virtual address.
+    '''
+
+
+class BoolEnableDisable(Enum):
```

Comment:
Another bool, there may be more.

Reply:
There were several. All removed.

Reply:
Actually, these are in a PR that hasn't been merged yet: #717